odbc_catalog.c

```diff
@@ -3834,9 +3834,33 @@ retrieve_table_from_db_class (int cci_connection, char *table_name, T_CCI_ERROR
   int cci_request;
 
   char *sql_statment = "SELECT class_name FROM db_class WHERE class_name = ?";
-  char *param_list[] = { table_name };
+  char tablename [512];
+  char *param_list[] = { tablename };
+  char query_buf [512];
```

Comment:
I think query_buf buffer size should be greater than tablename + owner + sql_statment.

Reply:
Thank you for your comment, I'll fix it.